utils/wrappers/oracleWrapper.ts

```diff
@@ -237,4 +262,48 @@ export class OracleWrapper {
 
     return priceArray;
   }
+
+  public async updateDriftlessHistoricalPriceFeedAsync(
+    dailyPriceFeed: DriftlessHistoricalPriceFeedContract,
+    medianizer: MedianContract,
+    price: BigNumber,
+    from: Address = this._contractOwnerAddress
+  ): Promise<void> {
+    await this._blockchain.increaseTimeAsync(ONE_DAY_IN_SECONDS);
+
+    const lastBlock = await web3.eth.getBlock('latest');
+    await this.updateMedianizerPriceAsync(
+      medianizer,
+      price,
+      lastBlock.timestamp + 1,
+    );
+
+    await dailyPriceFeed.poke.sendTransactionAsync(
+      { gas: DEFAULT_GAS},
+    );
+  }
+
+  public async batchUpdateDriftlessHistoricalPriceFeedAsync(
```

Comment:
You should be able to dedupe this by using the existing methods by simply modifying the input type

` dailyPriceFeed: (HistoricalPriceFeedContract  | DriftlessHistoricalPriceFeedContract)`

Reply:
The interfaces are different due to the timestamp naming. Tried it originally but was a no go.